Exiled.Events/Events.cs

```diff
@@ -153,10 +153,11 @@ public void Patch()
                 bool lastDebugStatus = Harmony.DEBUG;
                 Harmony.DEBUG = true;
 #endif
-                if (PatchByAttributes())
+                PatchByAttributes(out int failedPatch);
+                if (failedPatch == 0)
                     Log.Debug("Events patched successfully!");
                 else
-                    Log.Error($"Patching failed!");
+                    Log.Error($"Patching failed! There are {failedPatch} patch broken.");
```

Comment:
```suggestion
                    Log.Error($"Patching failed! There are {failedPatch} broken patches.");
```
Minor English fix :)